.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,30 @@
-### Context
+## Context
 
-### Changes proposed in this pull request
+<!-- Why are you making this change? What might surprise someone about it? -->
 
-### Guidance to review
+## Changes proposed in this pull request
 
-### Checklist
+<!-- If there are UI changes, please include Before and After screenshots. -->
 
+## Guidance to review
+
+<!-- How could someone else check this work? Which parts do you want more feedback on? -->
+
+## Link to Trello card
```

Comment:
I don't really find this useful since we have the trello integration that puts the ticket in the comments of the PR. I always skip it. Maybe this can be a checkbox in `Things to check`.

It's not a big deal though.

Reply:
I agree with this. I'd rather attach the PR to the Trello card, which then automatically links it to the PR. That's why it wasn't already on this checklist.

Reply:
Also agree. I never knew what people prefer so I always put both. " I'd rather attach the PR to the Trello card, which then automatically links it to the PR. That's why it wasn't already on this checklist." Agreed!

Reply:
The people have spoken 542a744043b2fa6f16871e5929f38ee99917069a & https://github.com/DFE-Digital/apply-for-teacher-training/pull/9620/commits/bd0edb6cd2f5f351b63e4aa78fffcfa2626636ac

Reply:
✊🏻